management-api-server/src/test/java/com/datastax/mgmtapi/BaseDockerIntegrationTest.java

```diff
@@ -32,6 +32,7 @@
 public abstract class BaseDockerIntegrationTest {
   protected static final Logger logger = LoggerFactory.getLogger(BaseDockerIntegrationTest.class);
   protected static final String BASE_PATH = "http://localhost:8080/api/v0";
+  protected static final String BASE_PATH_V2 = "http://localhost:8080/api/v2";
```

Comment:
thought:
This seems like we don't have any of the `v1` endpoints covered in the tests. I'll create a ticket to add them later.

Reply:
I've captured this in https://github.com/k8ssandra/management-api-for-apache-cassandra/issues/362